MapleServer2/Constants/Skills/SkillTreeOrdered.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System.Collections.Generic;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Constants.Skills
+{
+    public static class SkillTreeOrdered
```

Comment:
What happened to being able to send the skills in the order they are listed in the xml? Did it end up not working?

Reply:
it wasn't loading the second passive skill that all jobs have from default and didn't get the same result with using the `table/job` so instead i load the subskill list for each job that they are all there and in future use the `feature` and `subJobCode` to define when it is awakened or not and load the corresponding skill.